qiskit_experiments/curve_analysis/curve_data.py

```diff
@@ -25,94 +25,138 @@
 
 @dataclasses.dataclass(frozen=True)
 class SeriesDef:
-    """Description of curve."""
+    """A dataclass to describe the definition of the curve.
+
+    Args:
```

Comment:
I was curious what the "correct" way to comment a dataclass is and found this https://stackoverflow.com/a/51131941

They suggest preferring option (3) using `Attributes:` rather than using `Args:`. I didn't test myself, but I wonder if this will will override the empty attribute desciptions currently populated [here](https://qiskit.org/documentation/experiments/stubs/qiskit_experiments.curve_analysis.SeriesDef.html#qiskit_experiments.curve_analysis.SeriesDef). Checking the artifact on this PR the attribute doc stirngs are unchanged, but the `Parameters:` field is populated correctly from these args.

![image](https://user-images.githubusercontent.com/2235104/165178641-7b08800a-f09b-4954-b2c9-813bc76ddc1c.png)

Can you try changing to attributes and checking what the built docs looks like and pick the one you think looks best?

Reply:
Thanks for searching. I also prefer attribute style but I was not sure the best one. I just copied style from primitives in terra :)
![image](https://user-images.githubusercontent.com/39517270/165182038-e21864bb-bf4d-46a1-b2d2-593859e6ca08.png)